### ISSUE TITLE
Expand the width of Tooltip

### ISSUE THREAD

**Issue by riteshchandna** [xuopled/react-simple-tooltip#18]
Is there an option to expand the width of Tooltip? I want to display a long path and I am not able to customize the width of Tooltip. It's very small. Please have a look at the attached image. 

![tooltip_width](https://user-images.githubusercontent.com/10916624/53996416-497c4180-4106-11e9-9224-2c48be7ed68f.png)

**Comment by riteshchandna** [comment]
![expected_width](https://user-images.githubusercontent.com/10916624/53996496-9c55f900-4106-11e9-9481-3eb5a2ba01b4.png)

It would be great if I can achieve this functionality.

**Comment by cedricdelpoux** [comment]
Hello @riteshchandna, I just published a new version `v2.5.0`.
You can use the `style` prop or the `customCss` prop to prevent tooltip to break with white space.

```
<ReactSimpleTooltip
          content="😎"
          style={{whiteSpace: "nowrap"}}
        >
          <Zone>Hover me !</Zone>
</ReactSimpleTooltip>
```

```
import {css} from "styled-components"

<ReactSimpleTooltip
          content="😎"
          customCss={css`white-space: nowrap;`}
        >
          <Zone>Hover me !</Zone>
</ReactSimpleTooltip>
```

**Comment by iforwms** [comment]
Whilst this does work, it's only because it's cascading from the parent. 

Is there any way at the moment to add CSS to the tooltip itself? In my case I want the tooltip text to wrap, but set a `min-width`. At the moment both `customCss` and `style` props add CSS to the container div.

EDIT: As a side note I used the sibling selector in css to achieve what I needed.

**Normal**
```js
<Tooltip content="hello">
  <div class="tooltip">Content here</div>
</Tooltip>
```

**On hover**
```js
<Tooltip content="hello">
  <div class="tooltip">Content here</div>
  <div>hello</div>
</Tooltip>
```

```css
.tooltip + div {
  min-width: 120px;
}
```
